README.md

```diff
@@ -1 +1,81 @@
-# Connect-landing-page
\ No newline at end of file
+
+# Project Name: Smart Online Coding School
+
+>**Smart is a platform that provides student from all around the world an opportunity to learn coding without having to worry distance, Fees and other hinders that can easily be found in tradition school systems.(This is A capstone project done at Microverse).**
+
+## Built With
+
+- HTML and CSS
+- Javascript
+- npm(Node.js)
+- Font-awesome
+- SASS Preprocessor( SCSS )
+
+## Live Demo
+
+[**See Project 🚀**](https://davezag.github.io/Smart-Online-school-capstone-project/)
+
+<!-- 
+## Video Demonstration
+
+[**Watch**](https://) -->
+
+## Screenshots
+
+![Desktop Design](./preview/screenshot-127.0.0.1_5500-2022.09.15-16_47_38.png)
+ <p align="center">
+  <img src="./preview/screenshot-127.0.0.1_5500-2022.09.15-16_52_09.png">
+</p>
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- To have basic understanding of the web, and web technologies like HTML, CSS and Javascript.
+- Web browser: Chrome (Prefered one).
+
+### Setup
+
+- Here is a command to clone this project
+
+  `git clone https://github.com/DaveZag/Smart-Online-school-capstone-project.git`
+
+Once clone on your computer, open the folder then open the index.html file in your browser. If you opened the folder in vscode or another Code Editor, look for the extesion __"live server"__; install it then click on Go Live at the bottom right of you vscode editor and voila.
+
+## Author
+
+👤 **Dave ZAGABE**
+
+## Repo Links
+
+- Repository: [Smart](https://github.com/DaveZag/Smart-Online-school-capstone-project)
+
+## Socials
+
+>- GitHub: [@Dave Zag](https://github.com/DaveZag)
+>- Twitter: [@Dave Zagabe](https://twitter.com/davezagabe2)
+>- LinkedIn: [@Dave Zagabe](https://www.linkedin.com/in/dave-zagabe-03597a247/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Microverse](https://github.com/microverseinc)
+
```

Comment:
- [ ] Please give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in your acknowlegements, as required by the [Creative Commons License](https://creativecommons.org/licenses/).ives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Hi @Kamba56 I think that part is already done in the credit section in ReadME [Credit](https://github.com/DaveZag/Smart-Online-school-capstone-project/compare/main...home-page#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R75-R78).

Reply:
Hi @DaveZag, sorry about that.